android/lib/src/main/java/org/vosk/android/SpeechService.java

```diff
@@ -34,14 +40,20 @@ public class SpeechService {
     private final Recognizer recognizer;
 
     private final int sampleRate;
-    private final static float BUFFER_SIZE_SECONDS = 0.2f;
+    private final static float BUFFER_SIZE_SECONDS = 0.4f;
+    private final static int RECOGNIZER_QUEUE = 5;
     private final int bufferSize;
     private final AudioRecord recorder;
 
     private RecognizerThread recognizerThread;
 
+    private boolean accepted = false;
```

Comment:
you can clean this variable, we are using a local variable with the same name in the TranscriptionThread